server/datastore/mysql/migrations/tables/20240701113709_VPPDBUpdates.go

```diff
@@ -92,7 +89,8 @@ CREATE TABLE host_vpp_software_installs (
 	updated_at TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
 	PRIMARY KEY(id),
 	FOREIGN KEY (user_id) REFERENCES users (id) ON DELETE SET NULL,
-	FOREIGN KEY (adam_id) REFERENCES vpp_apps (adam_id) ON DELETE CASCADE
+	FOREIGN KEY (adam_id) REFERENCES vpp_apps (adam_id) ON DELETE CASCADE,
```

Comment:
I thought it was intended that when a VPP app is deleted from Fleet, existing installs are not [removed](https://www.figma.com/design/QsO3Q03QGVbiSknGpLqgAB/%2318867-Install-Apple-App-Store-apps-on-macOS?node-id=400-16974&m=dev)? Will this cascade cause any issues?

Reply:
@gillespi314 so in the actual delete flow, we do not delete from `vpp_apps`, just from `vpp_apps_teams`. So the data in this table wouldn't be deleted. That being said, this cascade _would_ cause the data in `host_vpp_software_installs` to be deleted if we actually deleted from the `vpp_apps` table. 

should we just remove it? cc @roperzh

Reply:
fwiw the equivalent column in the installers world also has on delete cascade

Reply:
Yeah, and there's also the [issue](https://github.com/fleetdm/fleet/issues/20271) of host deletions to think about.